tests/doc/parse.js

```diff
@@ -91,6 +91,24 @@ describe('custom string on node', () => {
     expect(str).toBe('a: \n')
     expect(YAML.parse(str)).toEqual({a: null})
   })
+
+  test('empty string camelBool', () => {
+    YAML.scalarOptions.bool.trueStr = 'True'
+    YAML.scalarOptions.bool.falseStr = 'False'
+    const doc = YAML.parse('[true, false]')
+    const str = YAML.stringify(doc)
+    expect(str).toBe('- True\n- False\n')
+    expect(YAML.parse(str)).toEqual([true, false])
+  })
+
+  test('empty string upperBool', () => {
+    YAML.scalarOptions.bool.trueStr = 'TRUE'
+    YAML.scalarOptions.bool.falseStr = 'FALSE'
+    const doc = YAML.parse('[true, false]')
+    const str = YAML.stringify(doc)
+    expect(str).toBe('- TRUE\n- FALSE\n')
+    expect(YAML.parse(str)).toEqual([true, false])
+  })
```

Comment:
What's the new ground being covered by these tests?

Reply:
This test is to check whether the program can output correctly after the user sets the scalarOptions.

Reply:
Okay, so they should then go with the stringify tests, rather than here.

Reply:
Good! done.